fixtures/jsii-calc/lib/function-returning-anonymous-class.ts

```diff
@@ -0,0 +1,16 @@
+type Constructor = { new (...args: any[]): {} };
+
+/**
+ * Just the mere presence of this function is enough to break jsii, even if it's not exported
```

Comment:
But it is exported. You mean, not used?

Reply:
I mean not exported from the jsii root module.